tests/sandbox/create_message_batches/test_field_validation.py

```diff
@@ -263,7 +311,19 @@ def test_data_too_few_items(nhsd_apim_proxy_url, property, pointer, correlation_
 @pytest.mark.parametrize("correlation_id", CORRELATION_IDS)
 def test_invalid_nhs_number(nhsd_apim_proxy_url, nhs_number, correlation_id):
     """
-    .. py:function:: Test invalid NHS numbers
+    .. py:function:: Scenario: A user submitting a request with an invalid \
+        NHS number receives a 400 'Invalid NHS Number' response
+
```

Comment:
link to docs where it explains what a valid/invalid NHS number is? we put it on a ticket recently

Reply:
Addressed in nhs_number.rst, linked below given/when/then